src/ImGui.NET/ImGuiNative.cs

```diff
@@ -71,10 +71,6 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern DrawList* igGetWindowDrawList();
         [DllImport(cimguiLib)]
-        public static extern NativeFont* igGetWindowFont();
```

Comment:
I'm assuming these were replaced directly by `igGetFont` and `igGetFontSize`. I wonder if it's worth leaving dummy versions of these around that are `[Obsolete]` and just throw an exception?